app/Repositories/Project/ProjectRepository.php

```diff
@@ -1074,4 +1090,51 @@ private function getActivityGrade($projectId, $activityParam)
         return null;
 
     }
+
+    public function getProjects($suborganization, $data) {
+
+        $authenticated_user = auth()->user();
+        $query = $authenticated_user->projects();
+
+//        if (isset($data['skipPagination']) && $data['skipPagination'] === 'true') {
+        if (!isset($data['size'])) {
+            return $query->where('organization_id', $suborganization->id)
+                ->whereNull('team_id')
```

Comment:
The where conditions below are common so can be moved out of if.

->where('organization_id', $suborganization->id)
                ->whereNull('team_id')